### DISCUSSION TITLE
Issues regarding working with a specific type of packages

### DISCUSSION THREAD

**Discussion by CanYing0913** [marcelotduarte/cx_Freeze#1841] [General]
Hi thank you for this wonderful project!
I just want you to know there is probably a potential bug as I wrote [here](https://github.com/imagej/pyimagej/issues/253). It seems the name tracking is lost? If you can take a look that is so much appreciated! :)
----
Edit: I think it is more of a runtime problem since running the script with the name "pyimagej" works fine.

**Comment by marcelotduarte** [comment]
Hi! When the package name differs od the import name and it depends on the metadata, we need to make a hook. You can move that issue to cx_Freeze, I'll make the hook.
